crates/bevy_scene/src/bundle.rs

```diff
@@ -0,0 +1,94 @@
+use std::ops::{Deref, DerefMut};
+
+use bevy_asset::Handle;
+use bevy_ecs::{
+    bundle::Bundle,
+    change_detection::ResMut,
+    entity::Entity,
+    prelude::{Changed, Component},
+    system::{Commands, Query},
+};
+use bevy_transform::components::{GlobalTransform, Transform};
+
+use crate::{DynamicScene, InstanceId, Scene, SceneSpawner};
+
+/// [`InstanceId`] of a spawned scene. It can be used with the [`SceneSpawner`] to
+/// interact with the spawned scene.
+#[derive(Component)]
+pub struct SceneInstance(InstanceId);
+
+impl Deref for SceneInstance {
+    type Target = InstanceId;
+
+    fn deref(&self) -> &Self::Target {
+        &self.0
+    }
+}
+
+impl DerefMut for SceneInstance {
+    fn deref_mut(&mut self) -> &mut <Self as Deref>::Target {
+        &mut self.0
+    }
+}
+
+/// A component bundle for a [`Scene`] root.
+///
+/// Once the scene is spawned, the entity will have a [`SceneInstance`] component.
+#[derive(Default, Bundle)]
+pub struct SceneBundle {
+    /// Handle to the scene to spawn
+    pub scene: Handle<Scene>,
+    pub transform: Transform,
+    pub global_transform: GlobalTransform,
+}
+
+/// System that will spawn scenes from [`SceneBundle`].
+pub fn scene_spawner(
```

Comment:
Can we merge this with `dynamic_scene_spawner`? I don't think these make a ton of sense as distinct systems.

Reply:
they can run in parallel, it depends if we want to promote many small systems or less bigger systems?

Reply:
Hmm. I think that in this case it's clearer to merge them, and the overhead will outweigh the parallelism because of how rarely active these will be.

Reply:
merged!